benches/README.md

```diff
@@ -0,0 +1,54 @@
+# Miden VM Hash Functions 
+In Miden VM, we make use of different hash functions. Some of these are "traditional" hash functions, like `Blake3`, which are optimized for out of STARK performance, while others are algebraic hash functions, like `Rescue Prime`, and are more optimized for a better performance inside the STARK. In what follows, we benchmark several such hash functions and compare against other constructions that are used by other proving systems. More precisely, we benchmark:
+
+## Rescue Prime
+As specified [here](https://eprint.iacr.org/2020/1143) and implemented [in](https://github.com/novifinancial/winterfell/blob/46dce1adf0/crypto/src/hash/rescue/rp64_256/mod.rs).
+
+## Rescue Prime Optimized
+As specified [here](https://eprint.iacr.org/2022/1577) and implemented in this crate.
+
+## BLAKE3
+As specified [here](https://github.com/BLAKE3-team/BLAKE3-specs/blob/master/blake3.pdf) and implemented in this crate.
+
+## SHA256
+As specified [here](https://csrc.nist.gov/csrc/media/publications/fips/180/4/final/documents/fips180-4-draft-aug2014.pdf) and implemented [in](https://github.com/novifinancial/winterfell/blob/46dce1adf0/crypto/src/hash/sha/mod.rs).
+
+## Poseidon
+As specified [here](https://eprint.iacr.org/2019/458.pdf) and implemented (without vectorized instructions) [in](https://github.com/mir-protocol/plonky2/blob/main/plonky2/src/hash/poseidon_goldilocks.rs).
```

Comment:
Let me know if it is what you were thinking.

Reply:
I think the formatting is still a bit off. You could probably use the following Markdown format:
```
* **Rescue Prime** as specified [here](https://eprint.iacr.org/2020/1143) and implemented [here](https://github.com/novifinancial/winterfell/blob/46dce1adf0/crypto/src/hash/rescue/rp64_256/mod.rs).
* **Rescue Prime Optimized** as specified [here](https://github.com/BLAKE3-team/BLAKE3-specs/blob/master/blake3.pdf) and implemented in this crate.
```
It will appear like this:

* **Rescue Prime:** as specified [here](https://eprint.iacr.org/2020/1143) and implemented [here](https://github.com/novifinancial/winterfell/blob/46dce1adf0/crypto/src/hash/rescue/rp64_256/mod.rs).
* **Rescue Prime Optimized:** as specified [here](https://github.com/BLAKE3-team/BLAKE3-specs/blob/master/blake3.pdf) and implemented in this crate.


Also, linking the word "in" looks a bit weird. Let's change it to "here"?

Reply:
Done, thank you for the suggestion!